Extension/src/LanguageServer/client.ts

```diff
@@ -1004,22 +1006,31 @@ class DefaultClient implements Client {
             let showIntelliSenseFallbackMessage: PersistentState<boolean> = new PersistentState<boolean>("CPP.showIntelliSenseFallbackMessage", true);
             if (showIntelliSenseFallbackMessage.Value) {
                 ui.showConfigureIncludePathMessage(() => {
-                    let learnMorePanel: string = "Configuration Help";
+                    let configJSON: string = "Configure (JSON)";
+                    let configUI: string = "Configure (UI)";
                     let dontShowAgain: string = "Don't Show Again";
                     let fallbackMsg: string = this.configuration.VcpkgInstalled ?
-                        "Update your IntelliSense settings or use Vcpkg to install libraries to help find missing headers." :
-                        "Configure your IntelliSense settings to help find missing headers.";
-                    return vscode.window.showInformationMessage(fallbackMsg, learnMorePanel, dontShowAgain).then((value) => {
+                        "Update your IntelliSense settings in JSON or UI. Or use Vcpkg to install libraries to help find missing headers." :
```

Comment:
I dont think we need to add "JSON or UI" in the message.  It's already on the buttons.